src/data/AmazonReviewDataContinuousTokenization.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+from pathlib import Path
+from typing import Optional
+
+from torch.utils.data import DataLoader
+from torch.utils.data.sampler import SubsetRandomSampler
+from torchtext.datasets import AmazonReviewPolarity
+import pytorch_lightning as pl
+import torch
+import numpy as np
+
+
+project_dir = Path(__file__).resolve().parents[2]
+
+
+class AmazonReviewFullDataModule(pl.LightningDataModule):
```

Comment:
How does this differ from AmazonReviewData.py? What's the use case

Reply:
Instead of tokenizing the whole dataset and saving it, it tokenizes each batch continuously
I think this makes it easier to test, but yeah the model still isn't learning anyway